source/index.html.md.erb

```diff
@@ -55,6 +55,10 @@ are still working for GDS as an organization.
 - [storing and querying logs](standards/logging.html)
 - [supporting a service](standards/supporting-services.html)
 
+### Miscellaneous
```

Comment:
I think we need a different heading here... :-) 
If we have other guidance in the pipeline that doesn't fit in manuals or standards, perhaps that will give us an idea for a new category title.